src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  selectedTodo: Todo | null,
+  setSelectedTodo: (todo: Todo | null) => void,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  selectedTodo,
+  setSelectedTodo,
+}) => (
+  <tr
+    data-cy="todo"
+    className={classNames({
+      'has-background-info-light': selectedTodo?.id === todo.id,
+    })}
+    key={todo.id}
+  >
```

Comment:
Remove the 'key' attribute from the 'tr' element, as it's not needed in this context. The parent component should handle the 'key' attribute when rendering a list of TodoItems.